File: VariablesYOperaciones/VisualBasic/EquivalenciasPies.vb
Imports System

Module Program
    Sub Main(args As String())
        Dim Pie, metros, Totalpul, yardas As Integer
        Dim pulgadas As Integer = 12
        Dim yarda As Integer = 3
        Dim UNApulgada As Double = 2.54
        Dim UNmetro As Integer = 100
        Dim centimetros As Double
        Console.Write("Ingrese una distancia medida en pies: ")
        Pie = Console.ReadLine()

        Totalpul = Pie * pulgadas
        yardas = Pie * yarda
        centimetros = Totalpul * UNApulgada
        metros = centimetros * UNmetro

        Console.WriteLine("Equivalente en pulgadas: " & Totalpul)
        Console.WriteLine("Equivalente en yardas: " & yardas)
        Console.WriteLine("Equivalente en centímetros: " & centimetros)
        Console.WriteLine("Equivalente en metros: " & metros)

        Console.ReadKey()
    End Sub
End Module

File: VariablesYOperaciones/C#/EquivalenciasPies.cs
using System;

namespace EquivalenciasPies
{
    public class Program
    {
        public static void Main(string[] args)
        {
            int Pie, Tpulg, yardas;
            byte pulgadas = 12;
            byte yarda = 3;
            double Apulgada = 2.54;
            byte Ametros = 100;
            double centimetros, metros;
            Console.Write("Ingrese una distancia medida en pies: ");
            Pie = Convert.ToUInt16(Console.ReadLine());
            Tpulg = Pie * pulgadas;
            yardas = Pie * yarda;
            centimetros = Tpulg * Apulgada;
            metros = centimetros * Ametros;
            Console.WriteLine("Equivalencia en Pulgadas: " + Tpulg);
            Console.WriteLine("Equivalencia en Yardas: " + yardas);
            Console.WriteLine("Equivalencia en Centímetros: " + centimetros);
            Console.WriteLine("Equivalencia en Metros: " + metros);
        }
    }
}